CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 0.13.0
```

Comment:
I think this is rather an addition to `0.12.3`? Or was this a mistake? Then let's rename `0.12.3` to `0.13.0`. In either case there shouldn't be two versions for a single release...

Reply:
OK👌